### DISCUSSION TITLE
Accessibility guidelines for VoiceOver and Talkback

### DISCUSSION THREAD

**Discussion by thisilver** [imaNNeo/fl_chart#1286] [General]
Hi there,

I'm currently designing a experience with Line Charts and would like to understand what are the guidelines for mapping interactions for the screen readers.

What level of details can I get with the default Line Charts?

Thanks,

**Comment by imaNNeo** [comment]
Hi there,

We don't support any built-in semantics at the moment,
btw, you can wrap any widget (such as [LineChart](https://github.com/imaNNeo/fl_chart/blob/master/repo_files/documentations/line_chart.md) or any other type of chart) with a [Semantics](https://api.flutter.dev/flutter/widgets/Semantics-class.html) widget to describe the information of chart.

About the interactions, I don't have any idea how we can handle them.
Can you help us?

Thanks for talking about this important topic.
